{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/components/util.jinja

```diff
@@ -1,7 +1,9 @@
 {% macro attributes(kwargs={}) %}
-    {% for name, value in kwargs.items() %}
-        {% if value %}
-            {{ name }}{% if value is not sameas True %}="{{ value }}"{% endif %}
-        {% endif %}
-    {% endfor %}
+    {% if kwargs != None %}
```

Comment:
Nice add here. I think you could simplify it to `{% if kwargs %}` if you wanted?